crates/iroha_data_model/src/visit.rs

```diff
@@ -373,7 +373,7 @@ leaf_visitors! {
     visit_set_parameter(&SetParameter),
     visit_execute_trigger(&ExecuteTrigger),
     visit_log(&Log),
-    visit_custom(&CustomInstruction),
+    visit_custom_instructions(&CustomInstruction),
```

Comment:
```suggestion
    visit_custom_instruction(&CustomInstruction),
```

shouldn't it be singular? like the name of the struct `CustomInstruction`